src/pcapi/flask_app.py

```diff
@@ -107,7 +107,12 @@ def before_request() -> None:
 
 @app.after_request
 def log_request_details(response: flask.wrappers.Response) -> flask.wrappers.Response:
-    duration = round((time.perf_counter() - g.request_start) * 1000)  # milliseconds
+    if "request_start" in g:
```

Comment:
C'est pas plutôt hasattr qu'il faut utiliser ?
```suggestion
    if hasattr(g, request_start):
```

Reply:
dixit [la doc flask](https://flask.palletsprojects.com/en/1.1.x/api/#flask.ctx._AppCtxGlobals) la version `"request_start" in g` semble celle correcte